oar/cli/cmd_stage_testing.py

```diff
@@ -0,0 +1,26 @@
+import click
+import logging
+from oar.core.jenkins_helper import JenkinsHelper
+from oar.core.const import *
+from oar.core.worksheet_mgr import WorksheetManager
+
+logger = logging.getLogger(__name__)
+
+@click.command()
+@click.pass_context
+def stage_testing(ctx):
```

Comment:
still need to add this cmd to `cmd_group.py`

Reply:
Updated: Commands:
  change-advisory-status     Change advisory status e.g.
  check-greenwave-cvp-tests  Check Greenwave CVP test results for all...
  create-test-report         Create test report for z-stream release
  drop-bugs                  Drop bugs from advisories
  push-to-cdn-staging        Trigger push job for cdn stage targets
  stage-testing              Trigger stage pipeline test
  take-ownership             Take ownership for advisory and jira subtasks
  update-bug-list            Update bug status listed in report, update...